src/evaluate.ts

```diff
@@ -0,0 +1,60 @@
+import { BinaryOperationVertex, ReturnVertex, VertexKind, Value, ControlVertex, LiteralVertex, DataVertex } from 'graphir';
+import { GraphInterpreter } from './interpreter'
+import { BinaryOperation } from '../../TS-Graph-Extractor/src/types'
```

Comment:
The graph extractor should be a dependency for the interpreter itself (maybe for testing and such only).
If we want to used these enums we should move them to the GraphIR project.